README.md

```diff
@@ -293,14 +313,14 @@ generate from your probes. It writes those incoming reports as JSON
 lines to a file. Start it like so:
 
 ```
-$ cd collectors/modality-probe-udp-collector
+$ cd ../collectors/modality-probe-udp-collector
 $ cargo install --path .
-$ cd ../../
+$ cd ../../modality-probe-capi
```

Comment:
Ditto above, confusing to be in the capi directory, let's just go back to the root

Reply:
:+1: